protostar/commands/test/cheatcodes/declare_cheatcode.py

```diff
@@ -6,7 +6,7 @@
 from starkware.python.utils import from_bytes
```

Comment:
Please add the following test case for `expect_events` cheatcode:
`test_data_transformer_works_in_unit_testing`